tradeexecutor/state/identifier.py

```diff
@@ -424,7 +434,7 @@ def get_liquidation_threshold(self) -> Percent:
         """What's the liqudation threshold for this leveraged pair"""
         assert self.kind.is_leverage()
         # Liquidation threshold comes from the collateral token
-        return self.quote.liquidation_threshold
+        return self.quote.liquidation_threshold / 100
```

Comment:
It's. better to convert this to 0...1 when the `qute.liquidation_threshold` is set. This way all percents in the system are in the same 0...1 notation.

Reply:
Good point, I've removed this workaround and fixed it directly in backend